### PR TITLE
Support for automatic indexing for gene IDs from genes.tsv and chemical.tsv if possible.

### PR DESCRIPTION
**Pull request by bsantan**
- Automatically uses gene ID in genes.tsv to assign node label, and checks for chebi ID in chemical.tsv or otherwise allows manual indexing.
- This always assumes PKL is kg-type

### CODE REVIEW

File: assign_nodes.py [hunk 1]
```diff
@@ -22,6 +22,14 @@
 pd.set_option('display.max_rows', None)
 
 
+#Added here to avoid circular import
+def get_label(labels,value,kg_type):
```
Comment (lagillenwater) [inline assign_nodes.py:26]: I got an error here: 

image

**Review by lagillenwater — COMMENTED** [review]
Found an bug in line 28 of assign nodes

File: assign_nodes.py (hunk 1 above)
Comment (bsantan) [reply assign_nodes.py:26]: I believe this is fixed. I didn't realize some ncbi gene ID's wouldn't be in PKL, which is hard to know what to index them to.

Comment (lagillenwater) [reply assign_nodes.py:26]: This works now. It seems the problem was with some pseudogenes. We should consider how to deal with this. My thinking is to allow the user to skip this gene if it is not represented. I'll make an issue.

**Review by lagillenwater — APPROVED** [review]

**Pull request merged**